### DISCUSSION TITLE
Query Performance: realtime.list_changes

### DISCUSSION THREAD

**Discussion by henrybeggin** [supabase/supabase#14300] [Questions] [answered]
Hi In reviewing the Query Performance tab i see that the following query is consuming 98% of the total query time, it is called 100 times more than any other query and tends to execute more slowly than all other top querys listed 

The Query is:
"select * from realtime.list_changes($1, $2, $3, $4)" 

This isn't a query i've explicitly written and i cannot find realtime.list_changes in the documentation, has anyone run into this query causing provisioning issues? Clearly its related to realtime but beyond that i cant seem to tell what it does - i'm perhaps not experienced enough to identity the root cause.

Any comments/advise massively appreciated

**Comment by GaryAustin1** [comment, marked as answer]
See this discussion towards the end.
https://github.com/supabase/realtime/issues/529

When you have realtime connected the realtime server polls the database every 100msec.

**Reply by henrybeggin** [reply]
Hi Gary, thank you for the response would you expect enabling "realtime" to cause AWS slowdowns from "depleting EBS balance"? Is there a minimum configuration when turning on this feature?
